### ISSUE TITLE
MathML rendering errors

### ISSUE THREAD

**Issue by zopyx** [Trimmarks/vivliostyle-formatter-issues#34]
See
[vivliostyle-output.pdf](https://github.com/vivliostyle/vivliostyle-formatter-issues/files/104871/vivliostyle-output.pdf)

I assume this is related to MathJax and possibly a MathJax error.

Input:

https://github.com/zopyx/css-paged-media-tutorial/blob/master/lesson-mathml-torture-test/index.html

**Comment by kwkbtr** [comment]
Right.
I added a script tag loading MathJax (of the version Vivliostyle Formatter is using)

``` html
<script src="https://cdn.mathjax.org/mathjax/2.6-latest/MathJax.js?config=TeX-MML-AM_CHTML"></script>
```

and opened the HTML file with Google Chrome, resulted in the same errors.
Could you report the problem to [MathJax](https://github.com/mathjax/MathJax/issues)?

**Comment by zopyx** [comment]
Done

2016-01-26 13:31 GMT+01:00 KAWAKUBO Toru notifications@github.com:

**Comment by zopyx** [comment]
The issue is still there in 2016.4

**Comment by skoji** [comment]
Should be fixed in version 2016.7
